src/components/StatusContainer.jsx

```diff
@@ -1,77 +1,122 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { navigate, Link } from '@reach/router'
 import { EOS_COMMENT, EOS_ATTACHMENT } from 'eos-icons-react'
 import { strip } from '../utils/filterText'
+import { Draggable, Droppable } from 'react-beautiful-dnd'
 import Vote from './Vote'
 
+function StoryCard(props) {

```

Comment:
Let's move the StoryCard component to its own file

Reply:
Always go for an isolation of components approach. Remember that the better controlled the environment of your applications the more control you will have to scale, debug, and test components and modules.

Reply:
Ok